generators/app/index.js

```diff
@@ -128,35 +134,48 @@ class ConsoleGenerator extends Generator {
 
     spinner.start()
 
-    spinner.text = 'Getting projects...'
+    spinner.text = 'Getting Projects...'
     const projects = (await this.sdkClient.getProjectsForOrg(orgId)).body
     spinner.stop()
 
-    const projectsList = projects.map(item => item.title)
+    // show projects by title and reverse order to show latest first
+    const projectsList = projects.map(item => item.title).reverse()
```

Comment:
note that .reverse() modifies the projects object here as well (in place reverse), FYI. Not sure if it affects things later

Reply:
ok thanks for the heads up it shouldn't change much but will double check

Reply:
ok applied reverse on a copy